src/steps/build.py

```diff
@@ -143,17 +143,19 @@ def runp(cmd):
     return 0
 
 
-def run_federalist_script(branch, owner, repository, site_prefix,
-                          base_url='', user_env_vars=[]):
+def run_build_script(branch, owner, repository, site_prefix,
```

Comment:
should this be named `has_build_script` to be consistent with `run_build_script` or vice versa?

Reply:
hmm ... there is nothing build related about the script other than it being in the build file.  I myself had the same question when renaming it. i have renamed it to has_build_script ... if we move it outside the build file we can change it then. i think you posing the same question as i had, the renaming it makes it more readable. thanks